conf/messages

```diff
@@ -952,6 +955,8 @@ property-details.multiple-disposals.guidance.li4=acquisition amount, plus improv
 property-details.multiple-disposals.guidance.p3=Contract exchange usually happens when you and the buyer or new owner swap contracts and the buyer pays a deposit.
 property-details.multiple-disposals.guidance.agent.p3=Contract exchange usually happens when your client and the buyer or new owner swap contracts and the buyer pays a deposit.
 property-details.multiple-disposals.guidance.trust.p3=Contract exchange usually happens when the trust and the buyer or new owner swap contracts and the buyer pays a deposit.
+property-details.multiple-disposals.guidance.personalRep.p3=Contract exchange usually happens when the trust and the buyer or new owner swap contracts and the buyer pays a deposit.
```

Comment:
it's talking about a trust, but the message key is for a personal rep. Same thing for the capacitor message below